tests/Psl/Fun/PipeTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\Tests\Fun;
+
+use PHPUnit\Framework\TestCase;
+use Psl\Fun;
+use Psl\Str;
+
+class PipeTest extends TestCase
```

Comment:
final. ( same for the other test )

Reply:
might be good to enforce this through cs